src/lib/schematics/collection.json

```diff
@@ -6,41 +6,37 @@
     "ng-add": {
       "description": "Adds Angular Material to the application without affecting any templates",
       "factory": "./install",
-      "schema": "./install/schema.json",
       "aliases": ["material-shell"]
     },
     // Create a dashboard component
     "dashboard": {
       "description": "Create a card-based dashboard component",
       "factory": "./dashboard/index",
-      "schema": "./dashboard/schema.json",
       "aliases": ["material-dashboard"]
     },
     // Creates a table component
     "table": {
       "description": "Create a component that displays data with a data-table",
       "factory": "./table/index",
-      "schema": "./table/schema.json",
       "aliases": ["material-table"]
     },
     // Creates toolbar and navigation components
     "nav": {
       "description": "Create a component with a responsive sidenav for navigation",
       "factory": "./nav/index",
-      "schema": "./nav/schema.json",
-      "aliases": [ "material-nav", "materialNav" ]
+      // TODO(paul): re-add `materialNav` alias if we have the latest schematics version that
```

Comment:
Could you use `devversion` in TODOs instead of `paul`?

Reply:
Sure. Wasn't really sure if `devversion` is more ambiguous than `paul`.